### PR TITLE
Triangular_solve function for the Jax frontend

### PR DESCRIPTION
**Pull request by kimbring2**
https://github.com/unifyai/ivy/issues/4385

I need to manually set the num_positional_args argument as 2 to make them pass the test. I put the argument of the frontend function as like the [official document](https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.linalg.triangular_solve.html).

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.
